ingest/src/main/java/com/microsoft/azure/kusto/ingest/ManagedStreamingIngestClient.java

```diff
@@ -214,23 +214,27 @@ public IngestionResult ingestFromStream(StreamSourceInfo streamSourceInfo, Inges
                         try {
                             OneApiError oneApiError = webException.getApiError();
                             if (oneApiError.isPermanent()) {
-                                log.error("Error is permanent, stopping.");
                                 throw e;
                             }
                         } catch (JSONException je) {
                             log.info("Failed to parse json in exception, continuing.", je);
                         }
                     }
 
-                    log.info(String.format("Streaming ingestion failed, trying again after sleep of %s", retry.nextTimeToSleep()), e);
+                    log.info(String.format("Streaming ingestion failed attempt %d", currentAttempt), e);
```

Comment:
Reminder to change this to `currentAttempt + 1` if we stick with 0-based indexing

Reply:
The question is, if I display it as "attempt 1" should the ClientRequestId also be 1 indexed? Because that feels wrong (the CRI is machine readable), but not having the same value between them also feels wrong..

Reply:
As discussed, no issue